starter-code/package.json

```diff
@@ -3,22 +3,31 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www",
+    "start": "nodemon ./bin/www",
+    "start-debug": "nodemon --inspect ./bin/www",
     "dev": "DEBUG=starter-code:* nodemon ./bin/www",
     "dev-windows": "nodemon ./bin/www"
   },
   "dependencies": {
+    "bcrypt": "^3.0.6",
```

Comment:
pls add nodemon here else it will not work for user who do not have nodemon pre installed